setup.py

```diff
@@ -14,7 +14,7 @@
     zip_safe=False,
     include_package_data=True,
     package_data={'': ['README.rst']},
-    install_requires=['django>=1.8.0', 'rq>=0.5.5'],
+    install_requires=['django>=1.8.0', 'rq>=0.5.5','raven>=6.1.0'],
```

Comment:
I don't think this should be a hard dependency, it isn't required by RQ itself...

Reply:
Agree. Will fix

Reply:
Do you think I should specify it as optional via extras_require. See http://peak.telecommunity.com/DevCenter/setuptools#declaring-extras-optional-features-with-their-own-dependencies ? I can instead locally import and in case of an exception write a blurb to the console explaining the situation. What do you think ?

Reply:
I'd go with extras_require, as for the message I think the import error should be descriptive enough, rq also doesn't do any special information in case it's missing.